app/src/community/java/com/twilio/video/app/data/api/AuthServiceRequestDTO.kt

```diff
@@ -0,0 +1,7 @@
+package com.twilio.video.app.data.api
+
+data class AuthServiceRequestDTO(
```

Comment:
Do we have an error response data class?

Reply:
That's what I'm working on now. I'm writing tests for all of the error scenarios and the error DTO is part of that.